src/alt-gosling-model/alt-text/text-tree.ts

```diff
@@ -259,20 +259,20 @@ function addEncodingDescriptions(track: AltTrackSingle | AltTrackOverlaidByMark
     
     if (track.alttype === 'single' || (track.alttype === 'ov-mark' && track.appearance.details.mark) || track.alttype === 'ov-data-ind') {
         mark = track.appearance.details.mark as string;
-        markText = markToText.get(mark) as string;
+        markText = callMarkToText(mark) as string;
```

Comment:
I think this type assertion is not needed since the return value of `callMarkToText` is always `string`.

```suggestion
        markText = callMarkToText(mark);
```